XML/TestCases/PerformanceTests.xml

```diff
@@ -11,7 +11,7 @@
       <param>testDuration=NTTTCP_RUNTIME_IN_SECONDS</param>
       <param>testType=tcp</param>
       <param>testConnections=NTTTCP_TCP_CONNECTIONS</param>
-      <param>ntttcpVersion=NTTTCP_VERSION</param>
+      <param>ntttcpVersion=master</param>
```

Comment:
is this planned change from 1.4.0 to master? I remember the master had missing implementation, so we stick with 1.4.0 version for a while. I need your confirmation switching to master is in our plan.

Reply:
There are some new in ntttcp which include that the retans_segs is shown in the output by default. https://github.com/microsoft/ntttcp-for-linux/commit/bceb216da69d8ca5737a94d6639b6287330251ca

The version 1.4.0 couldn't meet our needs. So I chang the version to master.